Gemfile.lock

```diff
@@ -23,6 +23,7 @@ PATH
       devise_invitable
       factory_bot_rails
       faker
+      geocoder
```

Comment:
we have a few projects that use geocoder that can be refactored now that rad_common will have the gem